src/Components/MailtoDropdown.js

```diff
@@ -37,19 +37,22 @@ export const MAILTO_HANDLERS = {
         countries: ['all'],
     },
     copymanually: {
-        onClick: (d) => {
-            const dismiss = () => render('', document.body, modal);
+        onClick: (d, createModal, onSuccess) => {
+            const dismiss = () => {
+                createModal(null);
+                if (onSuccess) onSuccess();
+            };
             const onInputClick = (e) => {
                 if (previous_active_element.id === e.target.id) return;
 
                 e.target.select();
                 e.target.focus();
                 previous_active_element = e.target;
             };
-            const modal = render(
+            createModal((state) => (
                 <IntlProvider scope="generator" definition={I18N_DEFINITION}>
                     <Modal
-                        positiveText={<Text id="ok" />}
+                        positiveText={t('ok', 'generator')}
```

Comment:
Does that make a difference?

Reply:
Yes, this throws an error (in Chrome), because `<Text />` is an object, not a string.